libs/sfm/bundler_common.cc

```diff
@@ -266,5 +266,50 @@ load_survey_from_file (std::string const& filename,
     in.close();
 }
 
+/* ---------------------- Feature undistortion -------------------- */
+
+namespace {
+    double distort_radius (double const r2, float const* params)
+    {
+        double coeff = 1.0 + r2 * params[0] + r2 * r2 * params[1];
```

Comment:
Can you add a comment what you're computing in this function? I don't understand it. According to this formula, we return `r^2 * (1 + k0 r^2 + k2 r^4)^2` which contains the radius to the power of 8?!?

Reply:
It simply computes the squared distorted radius. I added a comment to clarify this.